bzt/modules/java.py

```diff
@@ -249,9 +249,9 @@ def startup(self):
                 props.write("target_%s=%s\n" % (index, item.replace(os.path.sep, '/')))
 
         class_path = os.pathsep.join(self.base_class_path)
-        junit_command_line = ["java", "-cp", class_path, "taurusjunit.CustomRunner", self.props_file]
+        junit_cmd_line = ["java", "-cp", class_path, "-Djna.nosys=true", "taurusjunit.CustomRunner", self.props_file]
```

Comment:
If it's for Windows only, maybe add it conditionally?

Reply:
Actually no, people encountered it on linux too.

Reply:
You're likely to be right, I've found this: https://github.com/java-native-access/jna/issues/384